src/_locales/sk/messages.json

```
@@ -0,0 +1,338 @@
+    },
+    "popup_pause_title": {
+        "message": "Pauza [Space]"
+    },
+    "popup_play_title": {
+        "message": "Prehrať [Space]"
+    },
+    "popup_forward_title": {
+        "message": "Skok dopredu [F]"
+    },
+    "popup_next_title": {
+        "message": "Prejsť na ďalšie [PageDown]"
+    },
+    "popup_volume_title": {
+        "message": "$VALUE$% [-/+]",
+        "placeholders": {
+            "value": {
+                "content": "$1"
+            }
+        }
+    },
+    "popup_mute_title": {
+        "message": "Stíšiť / Obnoviť hlasitosť [F8]"
+    },
+    "popup_repeat_title": {
+        "message": "Opakovať"
+    },
+    "popup_shuffle_title": {
+        "message": "Náhodne"
+    },
+    "popup_clear_title": {
+        "message": "Vyčistiť"
+    },
+    "popup_contextmenu_title": {
+        "message": "Otvoriť kontextovú ponuku [C]"
+    },
+    "popup_up_title": {
+        "message": "Hore [Arrow Up]"
+    },
+    "popup_info_title": {
+        "message": "Zobraziť informácie [I]"
+    },
+    "popup_left_title": {
+        "message": "Vľavo [Arrow Left]"
+    },
+    "popup_select_title": {
+        "message": "Vybrať [Enter]"
+    },
+    "popup_right_title": {
+        "message": "Vpravo [Arrow Right]"
+    },
+    "popup_back_title": {
+        "message": "Späť [Backspace]"
+    },
+    "popup_down_title": {
+        "message": "Dole [Arrow Down]"
+    },
+    "popup_osd_title": {
+        "message": "Zobraziť OSD [M]"
+    },
+    "popup_home_title": {
+        "message": "Prejsť domov"
+    },
+    "popup_fullscreen_title": {
+        "message": "Vypnúť/zapnúť celú obrazovku [Tab]"
+    },
+    "popup_opensendtext_title": {
+        "message": "Poslať text"
+    },
+    "popup_playerprocessinfo_title": {
+        "message": "Zobraziť informácie o prehrávaní [O]"
+    },
+    "popup_empty_textcontent": {
+        "message": "Playlist prázdny"
+    },
+    "popup_playItem_title": {
+        "message": "Prehrať"
+    },
+    "popup_removeItem_title": {
+        "message": "Odstrániť"
+    },
+    "popup_text_placeholder": {
+        "message": "Text na odoslanie"
+    },
+    "popup_done_textcontent": {
+        "message": "Skončiť po odoslaní"
+    },
+    "popup_cancel_textcontent": {
+        "message": "Zrušiť"
+    },
+    "popup_sendtext_textcontent": {
+        "message": "Odoslať"
+    },
+    "popup_configure_textcontent": {
+        "message": "Nastaviť"
+    },
+    "popup_text_textcontent": {
+        "message": "Zmeniť server:"
+    },
+
+    "options_serverTitle_textcontent": {
+        "message": "Spojenie"
+    },
+    "options_serverIntro_textcontent": {
+        "message": "Aby ste mohli poslať videá/hudbu do Kodi, musíte najprv aplikáciam na iných systémoch povoliť diaľkové ovládanie. Táto voľba je v Kodi v sekcii Nastavenia / Servery / Ovládanie. Potom vyplňte adresu Kodi nižšie. Túto adresu nájdete v sekcii Nastavenia / Systémové informácie / Súhrn v Kodi."
```

Kodi uses [_Povoliť diaľkové ovládanie pomocou programov v iných systémoch_](https://github.com/xbmc/translations/blob/master/kodi-translations/kodi-main/merged-langfiles/kodi.core/resource.language.sk_sk/resources/strings.po#L2802) for _Allow remote control by programs on other systems_.

```suggestion
        "message": "Aby ste mohli poslať videá/hudbu do Kodi, musíte najprv musíte najprv Povoliť diaľkové ovládanie pomocou programov v iných systémoch. Táto voľba je v Kodi v sekcii Nastavenia / Servery / Ovládanie. Potom vyplňte adresu Kodi nižšie. Túto adresu nájdete v sekcii Nastavenia / Systémové informácie / Súhrn v Kodi."
```

---

It's pretty much the same. Kodi's translation sounds more like "allow remote control *by using* (with help of) programs on other systems" which I found not too accurate though.